docs/github-connector/configuration.md

```diff
@@ -0,0 +1,26 @@
+# Configuring GitHub Connector
+
+## Overview
+
+This document describes how to correctly connect GitHub repository to the GitHub Connector installed in Kyma environment. After completion you will be able to handle events incoming from GitHub in lambdas.
+
+## Prerequisites
+
+- Kyma with the GitHub Connector [installed](/docs/github-connector/installation.md)
+- Connection to Kyma
+
+## Installation
+
+1. Find the newly created github-connector application and [bind it to the namespace](https://kyma-project.io/docs/components/application-connector/#tutorials-bind-an-application-to-a-namespace) of your choice.
```

Comment:
Is `github-connector` the name of the app (file)? Or should it be `The GitHub Connector` application? Change if necessary.